extensions/wikia/TemplateClassification/TemplateClassification.hooks.php

```diff
@@ -228,4 +234,36 @@ private function getTemplateTypeForEdit( \Title $title, $wikiaId ) {
 
 		return $types;
 	}
+
+	/**
+	 * Prepare bulk template classification action and adds to possible action links.
+	 *
+	 * @param \Skin $skin
+	 * @param $links
+	 * @return bool
+	 */
+	public static function onSkinTemplateNavigation( \Skin $skin, &$links ) {
```

Comment:
These shouldn't be declared as static methods, as the hooks are being added to a class instance in the register method.